third_party/tsl/tsl/platform/cpu_info.h

```diff
@@ -148,6 +148,7 @@ enum CPUFeature {
   AARCH64_NEON = 1000,
   AARCH64_SVE = 1001,
   AARCH64_SVE2 = 1002,
+  AARCH64_BF16 = 48,  // BF16 on AArch64 systems
```

Comment:
Let's keep a running number?
```suggestion
  AARCH64_BF16 = 1003,  // BF16 on AArch64 systems
```